rocketchat-core/src/main/java/com/rocketchat/IRocketChatClient.java

```diff
@@ -0,0 +1,7 @@
+package com.rocketchat;
+
+import com.rocketchat.core.RocketChatClient;
+
+public interface IRocketChatClient {
```

Comment:
Unused file, please remove it.

Reply:
Actually you will require in future in android application class that implements IRocketChatClient. When you create instance of RocketChatClient inside application class,  you need to access it via interface. This interface will be needed when getting instance.

Reply:
It is not needed, we could use `RocketChatClient` directly, and mock it for tests...

And if we are going to define `RocketChatClient` as `final`, we should define a `RocketChatClient` `interface` with all the public entry points declared, and a internal `final` `RealRocketChatClient`